### ISSUE TITLE
Italian: repeated Vesper hymn

### ISSUE THREAD

**Issue by GoogleCodeExporter** [DivinumOfficium/divinum-officium#227]
```
http://divinumofficium.com/cgi-bin/horas/Pofficium.pl?date1=03-09-2014&command=p
rayVespera&version=Divino%20Afflatu&testmode=regular&lang2=Italiano&votive=

The hymn appears twice. Something to do with @ references?

```

Original issue reported on code.google.com by `igregord` on 10 Mar 2014 at 9:23

**Comment by rl1234567890** [comment]
I noticed that it does not seem to happen with the new German and Polish translations. I would like to know what should I do to fix the problem also for the Italian version.

**Comment by gregordick** [comment]
This was a bug in the logic for determining whether a hymn had been found.  Fix pushed as 56017ad.

**Comment by rl1234567890** [comment]
Fantastic! Thanks!
